tests/vcs/git/test_backend.py

```diff
@@ -75,3 +82,31 @@ def test_urlpathjoin(url: str, expected_result: str) -> None:
     path = "../other-repo"
     result = urlpathjoin(url, path)
     assert result == expected_result
+
+
+@pytest.mark.skipif(GIT_NOT_INSTALLLED, reason="These tests requires git cli")
```

Comment:
Do they? I thought they just use `dulwich` and do not need git installed (in comparison to `SystemGit`)?

Reply:
I thought the fixture uses `git cli` but now I see it's wasn't the case. Removed this `skipif`